src/App.tsx

```diff
@@ -53,7 +57,9 @@ const App = () => {
   }, []);
 
   useEffect(() => {
-    const barnMedLabels = mockPersonMedBarn.barn.map((barn) => {
+    let mapBarn = !erIDev && barneliste ? barneliste : mockPersonMedBarn.barn;
```

Comment:
Har dere vurdert å ha ett mocket API i stedet for å ha miljø-avhengig kode?

Reply:
Vi hadde det i starten, men det ble litt tungtvindt siden datamodellene våre ikke var helt klare ennå. 
Men er veldig for å ha det nå siden dataene er ganske klare og det hadde vært smuud! ✨

Reply:
Jepp, kan godt ta det i en refaktorering, men må først teste om denne fiksen funker.